blazor/diagram/ports/interaction.md

```diff
@@ -70,6 +71,97 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+
+<SfDiagramComponent @ref="diagram" Height="600px" Nodes="@nodes" DrawingObject="DrawingObject"/>
+
+@code
+{
+    DiagramObjectCollection<Node> nodes;
+
+    protected override void OnInitialized()
+    {
+        public SfDiagramComponent diagram;
+        nodes = new DiagramObjectCollection<Node>();
+        // A node is created and stored in nodes array.
+        Node node1 = new Node()
+        {
+            // Position of the node
+            OffsetX = 250,
+            OffsetY = 250,
+            // Size of the node
+            Width = 100,
+            Height = 100,
+            Style = new ShapeStyle() { Fill = "#6495ED", StrokeColor = "white" },
+            // Initialize port collection
+            Ports = new DiagramObjectCollection<PointPort>()
+            {
+                new PointPort()
+                {
+                    ID = "port1",
+                        Offset = new DiagramPoint() { X = 1, Y = 0.5 },
+                    Visibility = PortVisibility.Visible,
+                    //Set the style for the port
+                    Style= new ShapeStyle(){ Fill = "gray", StrokeColor = "black"},
+                    // Sets the shape of the port as Circle
+                    Width = 12, Height = 12, Shape = PortShapes.Square,
+                    // Enable drag operation for Port
+                    Constraints = PortConstraints.Default|PortConstraints.Draw
+                }
+            },
+        };
+        nodes.Add(node1);
+        Node node2 = new Node()
+        {
+            // Position of the node
+            OffsetX = 500,
+            OffsetY = 350,
+            // Size of the node
+            Width = 100,
+            Height = 100,
+            Style = new ShapeStyle() { Fill = "#6495ED", StrokeColor = "white" },
+            // Initialize port collection
+            Ports = new DiagramObjectCollection<PointPort>()
+            {
+                new PointPort()
+                {
+                    ID = "port1",
+                        Offset = new DiagramPoint() { X = 0, Y = 0.5 },
+                    Visibility = PortVisibility.Visible,
+                    //Set the style for the port
+                    Style= new ShapeStyle(){ Fill = "gray", StrokeColor = "black"},
+                    // Sets the shape of the port as Circle
+                    Width = 12, Height = 12, Shape = PortShapes.Square,
+                    // Enable drag operation for Port
+                    Constraints = PortConstraints.Default|PortConstraints.Draw
+                }
+            },
+        };
+        nodes.Add(node2);
+        DrawingObject = new Connector()
+        {
+            ID = "connector1",
+            Type = ConnectorSegmentType.Bezier,            
+        };
+    }
+}
+```
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/Ports/Interaction)
```

Comment:
You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/Ports/Interaction).